client/templates/course/lesson/quiz/questions/autoform/afQuestionOptionsGroup.js

```diff
@@ -0,0 +1,123 @@
+Template.afFieldQuestionOptionsGroup.helpers({
+
+    displayedOptions: function(){
```

Comment:
These helpers may be unnecessarily complicated. E.g. simply querying for question options, and omitting the `isCorrectAnswer` field would provide an array of question options to iterate through. Hence, the transformations would be unnecessary within the helper.